web/toolbar.js

```diff
@@ -270,10 +273,17 @@ class Toolbar {
     );
 
     const isDisable = mode === AnnotationEditorType.DISABLE;
-    editorFreeTextButton.disabled = isDisable;
-    editorHighlightButton.disabled = isDisable;
-    editorInkButton.disabled = isDisable;
-    editorStampButton.disabled = isDisable;
+    if (isDisable) {
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute#examples could we perhaps simplify this by using `toggleAttribute`?

Reply:
I didn't know this function.